charts/clusterpedia/templates/_helpers.tpl

```diff
@@ -361,3 +361,35 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- define "clusterpedia.hookJob.image" -}}
 {{ include "common.images.image" (dict "imageRoot" .Values.hookJob.image "global" .Values.global) }}
 {{- end -}}
+
+{{- define "clusterpedia.apiserver.tracing.env" -}}
+{{- if and (.Values.apiserver.telemetry.tracing.enabled) (.Values.apiserver.telemetry.tracing.addOtelEnv) }}
+- name: OTEL_EXPORTER_OTLP_ENDPOINT
+  value: {{ .Values.apiserver.telemetry.tracing.endpoint }}
+- name: 'OTEL_SERVICE_NAME'
+  value: {{ include "common.names.fullname" . }}-apiserver
+- name: 'OTEL_K8S_NAMESPACE'
+  valueFrom:
+    fieldRef:
+      apiVersion: v1
+      fieldPath: metadata.namespace
+- name: OTEL_RESOURCE_ATTRIBUTES_NODE_NAME
+  valueFrom:
+    fieldRef:
+      apiVersion: v1
+      fieldPath: spec.nodeName
+- name: OTEL_RESOURCE_ATTRIBUTES_POD_NAME
+  valueFrom:
+    fieldRef:
+      apiVersion: v1
+      fieldPath: metadata.name
+- name: OTEL_RESOURCE_ATTRIBUTES
+  value: 'k8s.namespace.name=$(OTEL_K8S_NAMESPACE),k8s.node.name=$(OTEL_RESOURCE_ATTRIBUTES_NODE_NAME),k8s.pod.name=$(OTEL_RESOURCE_ATTRIBUTES_POD_NAME)'
+{{- end -}}
+{{- end -}}
+
+{{- define "clusterpedia.apiserver.tracing.args" -}}
+{{- if .Values.apiserver.telemetry.tracing.enabled }}
+- --tracing-config-file=/etc/clusterpedia/trace/tracing-config.yaml
+{{- end -}}
```

Comment:
Since it is restricted to `Values.apiserver`, it is no longer generic and short, so this judgement logic can be written directly to apiserver.yaml